openshift/4.0/templates/logging/deploy-elasticsearch.yaml

```diff
@@ -116,10 +116,10 @@ objects:
   - kind: "PersistentVolumeClaim"
     apiVersion: v1
     metadata:
-      name: ${APP_NAME}-block-storage
```

Comment:
I agree that the other naming was a bit redundant (naming a route blah-route rather then blah seems a bit self-evident) but for this one there is a bit of value because block-storage is just one type of persistent volume claim. 

No strong feelings either way however.